dev-infra/pr/merge/pull-request.ts

```diff
@@ -114,16 +122,40 @@ export async function loadAndValidatePullRequest(
     hasCaretakerNote,
     targetBranches,
     title: prData.title,
-    commitCount: prData.commits,
+    commitCount: prData.commits.nodes.length,
   };
 }
 
+/* GraphQL schema for the response body the requested PR. */
+const PR_SCHEMA = {
+  url: graphQLTypes.string,
+  number: graphQLTypes.number,
+  commits: params({first: 100}, {
+    nodes: [{
+      commit: {
+        status: {
+          state: graphQLTypes.oneOf(['FAILURE' as const, 'PENDING' as const, 'SUCCESS' as const]),
```

Comment:
Can we say `as const` at the end?
```suggestion
          state: graphQLTypes.oneOf(['FAILURE', 'PENDING', 'SUCCESS'] as const),
```

Reply:
Ahh, didn't know that you could do that.  Done.